docs/ppp/vision-21112022.md

```diff
@@ -0,0 +1,85 @@
+# Sex is great, but have you tried noncustodial finance?
+
+Now that we have your attention, let us recap the two rules in Bitcoin:
+
+![not-your-keys.png](../assets/2022-11-21-not-your-keys.png)
+
+That is it, it is that dead simple: if you keep your Bitcoin on an exchange instead of putting them into self custody, you are going to have a bad time.
+Remember MtGox?
+
+> "That's too long ago, exchanges got better", John Doe, holds his Bitcoin on FTX.
+
+John, we are sorry to tell you: "Not your keys, not your coins!".
+Did you learn your lesson?
+
+> "But, but, I wanted to trade!", John Doe
+
+That is a common response we get, and we hear you!
+
+This is why we are bringing noncustodial peer-to-peer financial products to Lightning.
+
+![2022-11-21-goosbumps.png](../assets/2022-11-21-goosbumps.png)
+
+## The Vision
+
+Imagine a world just days before [hyperbitcoinization](https://bitcoinmagazine.com/hyperbitcoinization) finally hits.
+You walk along the beach enjoying the warmth of the sunset.
+
+You stop at a small food truck selling you a juicy steak, medium rare without sides, just perfect.
+You tap with your phone to pay with Lightning and sit down to enjoy the orange light of the sunset.
+
+Someone sells you a Tequila Sunrise - ahm - Tequila Sunset, freshly prepared right in front of you.
+
+While you take a sip from the drink, you read the latest news of the fed raising interest rates while throwing on the money printer at the same time to fight inflation.
+You casually take out your phone and open a long position on Bitcoin, knowing, that the last actions of the government will push the price even higher.
+
+Relaxed you lay back and open a streaming service which charges per seconds paid via Lightning.
+You wanted to check in on your favorite team, El Salvadore, who very likely will become world champion this year.
+Watching the team in orange shirts pushes your adrenaline higher and makes you euphoric.
```

Comment:
uber nit: I know it's a reference to being orange-pilled, but this is not their national shirt colours :P 
maybe we could change the name of the team?